src/server/pagasus_manual_compact_service.cpp

```diff
@@ -0,0 +1,271 @@
+// Copyright (c) 2017, Xiaomi, Inc.  All rights reserved.
+// This source code is licensed under the Apache License Version 2.0, which
+// can be found in the LICENSE file in the root directory of this source tree.
+
+#include "pagasus_manual_compact_service.h"
```

Comment:
我建议还是按照google的cpp编码规范来: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes
*Use standard order for readability and to avoid hidden dependencies: Related header, C library, C++ library, other libraries' .h, your project's .h.*